Naming.md

```diff
@@ -6,7 +6,15 @@ In general, follow the [official guidelines](https://swift.org/documentation/api
 
 ### Naming a file
 
-There's no three-letter prefix in Swift file naming. For example, if an Objective-C file named XYZEvent.m is to be converted to Swift, it should be renamed to Event.swift.
+- All Swift files should use the `.swift` extension.
+- The name of a file should make it clear what the contents are. For example, if there is a single class named `EasyTapButton` then the file should be called `EasyTapButton.swift`. If you have a view controller, along with some helper types, then using the name of the view controller will likely be the best choice. A good heuristic to use is that if it's ever unclear what to name a file, it may be worth splitting that file up.
+- There's no three-letter prefix in Swift file naming. For example, if an Objective-C file named XYZEvent.m is to be converted to Swift, it should be renamed to Event.swift.
+- If a file contains only one or more extensions, then the file name should be `MyType+Extensions.swift` unless the extension have a specific name; in that case the extension should be named `MyType+ExtensionName.swift`.
+
+### Naming a class, a struct or a protocol
+
+- Naming should be `PascalCase`
```

Comment:
I'd never heard this called PascalCase, but apparently it's a Microsoft-standard term? https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-1.1/x2dbyw72(v=vs.71)?redirectedfrom=MSDN

Reply:
I think it's relatively common? This came from the old Acompli style guide which was in use before Microsoft aquired it.

Reply:
As blew references:

- https://en.wikipedia.org/wiki/Camel_case
- https://swift.org/documentation/api-design-guidelines/#follow-case-conventions

I'd like call it `UpperCamelCase` too.

Reply:
I'll rename this to be UpperCamelCase to be consistent with Swift's standard.